test/e2e/common/tektonconfigdeployment_test.go

```diff
@@ -21,34 +21,48 @@ package common
 import (
 	"context"
 	"os"
+	"reflect"
 	"testing"
+	"time"
 
 	"github.com/google/go-cmp/cmp"
 	"github.com/tektoncd/operator/pkg/apis/operator/v1alpha1"
+	"github.com/tektoncd/operator/pkg/reconciler/kubernetes/tektonpipeline"
+	"github.com/tektoncd/operator/pkg/reconciler/kubernetes/tektontrigger"
 	"github.com/tektoncd/operator/test/client"
 	"github.com/tektoncd/operator/test/resources"
 	"github.com/tektoncd/operator/test/utils"
 	"github.com/tektoncd/pipeline/test/diff"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"knative.dev/pkg/ptr"
 )
 
-// TestTektonPipelinesDeployment verifies the TektonPipelines creation, deployment recreation, and TektonPipelines deletion.
+// TestTektonConfigDeployment does following checks
+// - make sure TektonConfig is created if AUTOINSTALL_COMPONENTS is true
+// - waits till TektonConfig ready status becomes true
+// - in case of OpenShift, runs rbac and addon test
+// - changes profile from all to basic and validates changes
+// - deletes a components and make sure it is recreated
+// - updates field in TektonConfig Spec and make sure they are reflected in configmaps
+// - Deletes and recreates TektonConfig and validates webhook adds all defaults
 func TestTektonConfigDeployment(t *testing.T) {
 	clients := client.Setup(t)
 
 	crNames := utils.ResourceNames{
-		TektonConfig: v1alpha1.ConfigResourceName,
-		Namespace:    "tekton-operator",
+		TektonConfig:    v1alpha1.ConfigResourceName,
+		Namespace:       "tekton-operator",
+		TargetNamespace: "tekton-pipelines",
 	}
 
 	platform := os.Getenv("TARGET")
 
 	if platform == "openshift" {
 		crNames.Namespace = "openshift-operators"
+		crNames.Namespace = "openshift-pipelines"
```

Comment:
I see `crNames.Namespace` same only values are changing :thinking:

Reply:
ah that's why test failing ! failed to push some changes ...